driver/csiplugin/connectors/connectors.go

```diff
@@ -96,6 +96,9 @@ const (
 	UserSpecifiedVolDirPath     string = "volDirBasePath"
 	UserSpecifiedNodeClass      string = "nodeClass"
 	UserSpecifiedPermissions    string = "permissions"
+	UserSpecifiedCompression    string = "compression"
+	UserSpecifiedReplication    string = "replication"
+	UserSpecifiedTier           string = "tier"
```

Comment:
Should we maybe think about naming this "storagePool" or similar to keep it related to GPFS convention?